src/Eccube/Form/Type/Admin/MasterdataDataType.php

```diff
@@ -56,7 +56,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     )),
                     new Assert\Regex(array(
                         'pattern' => '/^\d+$/u',
-                        'message' => 'form.type.numeric.invalid',
+                        'message' => $app->trans('form.type.numeric.invalid'),
```

Comment:
If you fix another problem exclude this issue, please commnet on this pullrequest.